### ISSUE TITLE
type JSONService has no field list_backups

### ISSUE THREAD

**Issue by phelipe** [JuliaCloud/AWS.jl#345]
I am trying to use AWS.jl to communicate to Dynamo.

```julia
using AWS
using AWS.AWSServices:dynamodb

my_config = AWSConfig(profile="default", region="us-west-2")
global_aws_config(my_config)

dynamodb.list_backups()
```
But I always get a error message:

```julia
julia> dynamodb.list_backups()
ERROR: type JSONService has no field list_backups
Stacktrace:
 [1] getproperty(x::JSONService, f::Symbol)
   @ Base ./Base.jl:33
 [2] top-level scope
   @ REPL[14]:1
```
Is this a bug or my code is wrong?

**Comment by mattBrzezinski** [comment]
TL;DR: The following code will fix it for you:

```julia
using AWS
@service DynamoDB

DynamoDB.list_backups()
```

---

The long-form answer. `AWS.jl` has two ways of making requests which I call `low-level` and `high-level`. The low-level requests require you to know *how* to make the request itself. DynamoDB uses JSON as a request type, which doesn't make the best example. So we'll do something that's a bit more "complex", getting objects from an S3 bucket.

To perform this low-level request,

```julia
using AWS.AWSServices: s3
bucket = "my-bucket"

s3("GET", "/$(bucket)")
```

There are two pieces of information you need to know here, the type of REST request to make (`GET`), and the endpoint itself `/$(bucket)`. Most people wouldn't care to know these two things for every request that they want to make, so there is a simplified version, the high-level requests.

For these high-level requests, all you need to know is *what* you want to achieve,

```julia
using AWS
@service S3
bucket = "my-bucket"

S3.list_objects(bucket)
```

I highly recommend using these high-level requests as they will make your life more simple. There are times where they will not work however, see #219 and for the time being you'll need to default back to the low-level requests. However, for 99% of things you will want to do the high-level ones will work.

**Comment by phelipe** [comment]
Thank you! :D

Issue solved
